src/app/content/studyGuides/selectors.ts

```diff
@@ -19,51 +24,78 @@ export const studyGuidesSummary = createSelector(
   (state) => state.summary
 );
 
-export const studyGuidesSummaryIsNotEmpty = createSelector(
+export const hasStudyGuides = createSelector(
   studyGuidesSummary,
-  (summary) => summary !== null
-    && summary.countsPerSource
-    && Object.keys(summary.countsPerSource).length > 0
+  (summary) => summary.totalCountsPerPage
+    && Object.keys(summary.totalCountsPerPage).length > 0
+);
+
+export const totalCountsPerPageOrEmpty = createSelector(
+  studyGuidesSummary,
+  (summary) => summary.totalCountsPerPage || {}
 );
 
 export const studyGuidesOpen = createSelector(
-  localState,
-  (state) => state.open
+  studyGuidesSummary,
+  (summary) => summary.open
 );
 
-export const studyGuidesIsLoading = createSelector(
-  localState,
-  (state) => state.loading
+export const summaryIsLoading = createSelector(
+  studyGuidesSummary,
+  (summary) => summary.loading
 );
 
-// Temporary to make hasMoreResults works
-export const studyGuidesPagination = createSelector(
-  localState,
-  (_state) => false
+export const summaryStudyGuidesPagination = createSelector(
+  studyGuidesSummary,
+  (summary) => summary.pagination
 );
 
-export const hasMoreResults = createSelector(
-  studyGuidesPagination,
-  (pagination) => Boolean(pagination)
+export const studyGuidesLocationFilters = createSelector(
```

Comment:
This selector is equal to `highlightLocationFilters`, we could move it to the the `content/selectors` and reference it as `parentSelectors.locationFilters`, but this would mean that we should also move `getHighlightLocationFilters` into `content/utils`
I personally was thinking about moving all type, interfaces, and utils which are used between MH and SG into `/content/utils/` and `/content/types` but it might be not worth of time

Reply:
See https://github.com/openstax/rex-web/pull/684#discussion_r441657225